### PR TITLE
Update recipe_instructions.Rmd

### PR DESCRIPTION
**Pull request by BB1464**
Fix typo

### CODE REVIEW

File: episodes/recipe_instructions.Rmd [hunk 1]
```diff
@@ -22,7 +22,7 @@ exercises: 2
 
 **Mix the dough:** 
 
-- Use a box grater to shred the cold butter into a large mixing bowl.
+- Use a box greater to shared the cold butter into a large mixing bowl.
```
Comment (hobytodges) [inline episodes/recipe_instructions.Rmd]:
```suggestion
- Use a box greater to shred the cold butter into a large mixing bowl.
```

**Review by hobytodges — CHANGES_REQUESTED** [review]
Thanks @BB1464 for contributing. I have suggested one change, but otherwise this looks ready to go. Please accept the change and then I can approve and merge.

**Review by kmadzima — APPROVED** [review]
Thanks a lot KM for proposing the changes.

**Comment by BB1464** [comment]
Thanks #1

**Pull request closed without merging**